web_external/Datasets/uploadWidget.styl

```diff
@@ -30,17 +30,22 @@
       background #6cc8a7
       box-shadow inset 0 3px 5px rgba(0,0,0,0.125);
 
+    &.g-drop-zone
```

Comment:
What's this selector do? The parent is also `.g-drop-zone`, so does it compile to be `.g-drop-zone.g-drop-zone`?